WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/FormattableAmountTextField.swift

```diff
@@ -31,15 +41,21 @@ struct FormattableAmountTextField: View {
                 .lineLimit(1)
                 .frame(maxWidth: .infinity, alignment: style.textAlignment)
                 .padding(5)
-                .if(focusAmountInput && style.showsBorder, transform: { field in
+                .if(focusAmountInput.wrappedValue && style.showsBorder, transform: { field in
                     field.roundedBorder(cornerRadius: 8, lineColor: Color(.wooCommercePurple(.shade60)), lineWidth: 1)
                 })
                 .onTapGesture {
-                    focusAmountInput = true
+                    focusAmountInput.wrappedValue = true
                 }
         }
         .fixedSize(horizontal: false, vertical: true)
     }
+
+    func focused(_ focusBinding: FocusState<Bool>.Binding) -> FormattableAmountTextField {
+        var copy = self
+        copy.externalFocusBinding = focusBinding
+        return copy
+    }
```

Comment:
I'm not sure to understand the usage of computed `focusAmountInput` with re-declaring `focused(...)` and differentiating internal from external focus input here (so the commit https://github.com/woocommerce/woocommerce-ios/pull/14902/commits/990053d15c04dc5e7c93f04ee3ca842384ae1596 ) 🤔

We seem to use the default SwiftUI's `focused<Value>(_ binding: FocusState<Value>.Binding, ... ` and commenting this out seems to work just fine, so we could remove these as well and be enough to set `isTextFieldFocused = false` as needed?

Reply:
Thanks, you're right! 💯 

I didn't think the default `.focused` would work when passing `.focused` through outside and within `FormattableAmountTextField` at the same time but I was wrong.